planner/core/planbuilder.go

```diff
@@ -1002,6 +1002,13 @@ func buildTableRegionsSchema() *expression.Schema {
 	return schema
 }
 
+func buildSplitRegionsSchema() *expression.Schema {
+	schema := expression.NewSchema(make([]*expression.Column, 0, 10)...)
```

Comment:
why making the slice with `10` capacity?

Reply:
Sorry, It's a mistake, already fixed.